README.md

```diff
@@ -939,7 +939,13 @@ class AuthorsNewPage < AePageObjects::Document
 end
 ```
 
-Additionally, if you can use `:is` to specify the collection type:
+
+#### Custom Collections
+
+Sometimes, we'll want a collection that supports more methods
+than what the [default implementation](https://github.com/appfolio/ae_page_objects/blob/master/lib/ae_page_objects/elements/collection.rb)
```

Comment:
Change this to a relative link:
```
[default implementation](./lib/ae_page_objects/elements/collection.rb)
```
rather than hardcoding it to link to `master`